editor/scene_tree_dock.cpp

```diff
@@ -1793,6 +1794,81 @@ void SceneTreeDock::_node_strip_signal_inheritance(Node *p_node) {
 	}
 }
 
+void SceneTreeDock::_node_strip_resource_reference(Node *p_node) {
+	List<PropertyInfo> props;
+	p_node->get_property_list(&props);
+
+	for (const PropertyInfo &E : props) {
+		if (!(E.usage & PROPERTY_USAGE_STORAGE) || E.type != Variant::OBJECT || E.hint != PROPERTY_HINT_RESOURCE_TYPE) {
+			continue;
+		}
+
+		Variant v = p_node->get(E.name);
+
+		if (v.is_ref_counted()) {
+			Ref<Resource> res = v;
+			static Vector<String> unique_exceptions = { "Image", "Shader", "Mesh", "FontFile" };
```

Comment:
This code unlinks the node from original scene, so there should be no exceptions. However you should make unique only Resources that are local to the scene (you can determine them with `is_built_in()` method).